learn/validator-trust-score.md

```diff
@@ -0,0 +1,273 @@
+
+</Callout>
+
+<details>
+
+<summary children:m-0>
+
+Details on how to calculate $C_i$
+
+</summary>
+
+$C_i$ is the number of blocks produced (created) by a validator in the epoch $i$:
+
+$$
+C_i = \sum_{j=0}^{N-1} c_j \quad \text{for } i = 0, 1, 2, \ldots, m-1
+$$
+
+$c_j$ is the number of blocks that the validator produced in the batch $j$, where $j \in [0, N-1]$.
+
+- $N$ is the number of batches in an epoch that can be retrieved from the [policy](htts://github.com/nimiq/core-rs-albatross/blob/albatross/primitives/src/policy.rs).
+- The number of blocks that the validators produced can be fetched from the blockchain via the rewarded inherent of a batch.
+
+</details>
+
+<details>
+
+<summary children:m-0>
+
+Details on how to calculate $H_i$
+
+</summary>
+
+$H_i$ is the likelihood that a validator will produce a block in the epoch $i$:
+
+$$
+H_i = \frac{h_{i,v}}{\sum_{k=0}^{V_i-1} h_{i,v}} \quad \text{for } i = 0, 1, 2, \ldots, m-1
+$$
+
+- $V_i$ is the number of active validators in the epoch $i$.
+- $h_{i,v}$ is the slot number for the validator $v$ in the epoch $i$.
+
+</details>
+
+$$
+r_i = \frac{C_i}{H_i}, \quad \text{for } i = 0, 1, 2, \ldots, m-1
+$$
+
+where $r_0$ is the Reliability value of the most recent epoch and $r_{m-1}$ is the Reliability of the oldest epoch.
+
+To combine all the Reliability scores into a single value, we do a moving average, where more recent epochs have higher weights than older ones.
+
+$$
+\bar{R} = \frac{\sum_{i=0}^{m-1} \left( 1-a\frac{i}{m-1} \right) r_i}{\sum_{i=0}^{m-1} \left( 1-a\frac{i}{m-1} \right)}, \quad a = 0.5
+$$
+
+Being $a$, the parameter determining how much the observation of the oldest epoch is worth relative to the observation of the newest epoch.
+
+#### Adjusting for High-Reliability Expectations
+
+The previous formula provides a weighted moving average score for validators reliability in block production, where a score of $0.9$ indicates a significant downtime of 10%[*](#notice-0.9), highlighting recent performance and the need for improved consistency.
+
+To better reflect the high standards required, we will plot the value on a circle, with $c$ as the parameter defining the slope of the arc.
+
+$$
+R=-c+1-\sqrt{-\bar{R}^{2}+2c\bar{R}+\left(c-1\right)^{2}}
+$$
+
+$$
+\text{Center of circle at (c,-c+1), where }
+c=-0.16
+$$
+
+<figure>
+
+<iframe src="https://www.desmos.com/calculator/zqemsh7yay?embed" aspect-video frameborder="0" allowfullscreen></iframe>
+
+<figcaption>
+
+Graph of the reliability score adjustment. The x-axis represents the reliability score, and the y-axis represents the adjusted reliability score.
+
+</figcaption>
+
+</figure>
+
+What we achieve with this adjustment is to penalise more severely those validators that have a low reliability score and are unable to produce blocks when expected.
+
+<small text-12 id="notice-0.9">
```

Comment:
I like this caption more than the one above next to the first image. What do you think?

Reply:
I need to work on the styles a bit more in the typography module. I will take into consideration your comment and update the whole design system. I like it more :)